### DISCUSSION TITLE
Simulate simultaneous send by multiple nodes

### DISCUSSION THREAD

**Discussion by Laxus-djm** [omnetpp/omnetpp#1152] [Q&A]
Hello everyone.  I have a scenarion where there are more than 10 or 20 nodes that frequently (let's say every 500ms) send some sort of beacon to specific node that act as a controller. These beacons should be sent at the same time. I am already using Csma to avoid colision, but the packets are taking longer time to reach the gcs since every node should wait for its turn . My question is : is there any possible way to apply techniques  such as FDMA  or TDMA or something similar to make sending simultaneous packets possible ?

**Comment by levy** [comment]
The INET WiFi model does this by filtering out received signals based on their channel (center frequency, bandwidth).
